Dockerfile

```diff
@@ -1,26 +1,17 @@
-FROM debian:sid
```

Comment:
You can use `openjdk-jre` or like I does to make the image smaller the alpine one.

The advantage is that you don't have to configure the JRE!